m4_sessions/1810-inning/public_data.md

```diff
@@ -1,21 +1,22 @@
 ## Group Members:
+* Angela Duran
+* Colin Armstrong
+* Matt Bricker
+* Michael Lyons
 
 
 
-## Name of Session: 
+## Name of Session: Public Data: Our Lord and Savior
```

Comment:
Tell me more about this title?  When I read it, I feel initially concerned that this might be offensive to someone who is religious